### DISCUSSION TITLE
Disable zoom animation

### DISCUSSION THREAD

**Discussion by OlaRune** [pelicanmapping/osgearth#2234] [Support Q&A] [answered]
Hello!

I have increased the zoom sensitivity on the EarthManipulator by using the setting `setScrollSensitivity(3.0)`. Now I would like to get the effect of snappier zoom by not animating the zoom effect, and instead just render the target zoom level in a single frame. How can I achieve this effect?

**Comment by gwaldron** [comment, marked as answer]
The setup is a little quirky. You need a very small (but non-zero) duration with a high sensitivity:
```c++
    auto man = new EarthManipulator();
    EarthManipulator::ActionOptions ao;
    ao.add(EarthManipulator::OPTION_DURATION, 0.001);
    man->getSettings()->bindScroll(EarthManipulator::ACTION_ZOOM_IN, osgGA::GUIEventAdapter::SCROLL_DOWN, 0, ao);
    man->getSettings()->bindScroll(EarthManipulator::ACTION_ZOOM_OUT, osgGA::GUIEventAdapter::SCROLL_UP, 0, ao);
    man->getSettings()->setScrollSensitivity(200.0);
```

**Reply by OlaRune** [reply]
This works well for a snappier zoom. Thanks!
